Chinese video ad filtering

```diff
@@ -0,0 +1,147 @@
+! Version: 202105021
+! Expires: 2 days
+! Homepage: http://bbs.kafan.cn/thread-1866845-1-1.html
+
+v.youku.com##.control-phonewatch,#right-title-ad-banner
+youku.com##+js(aeld,dblclick,ykplayer.switchFullscreen)
+!v.youku.com##+js(nano-stb)
```

Comment:
These rules are not supported on AdGuard.
Please use `#%#` or `#%#//scriptlet()`.

If you want to get more information, please read [this document](https://kb.adguard.com/en/general/how-to-create-your-own-ad-filters).